### DISCUSSION TITLE
Any way to enable double vpn?

### DISCUSSION THREAD

**Discussion by Ansem93** [bubuntux/nordvpn#225] [Q&A]
I found out that nordvpn support the double-vpn 
https://nordvpn.com/it/features/double-vpn/
There is some way to enable it? 
Would it be possible to have this? Or it is the obfuscate variable? 
If it's the obfuscate variable, would it be possible to specify in the description that it's the double-vpn?

**Comment by bubuntux** [comment]
read the doc, you can use 
`docker run --rm ghcr.io/bubuntux/nordvpn n_groups` and you will find the Double_VPN group, so you can connect using that group, again.. read the documentation
